docs/declarative-customization/list-form-conditional-show-hide.md

```diff
@@ -0,0 +1,110 @@
+
+- Open an item to view the item details in the display form
+
+3. If you are in a document library:
+
+- Select a file
+- In the **Information Pane**
+    - Locate the **Properties** section
+    - Click **Edit all**
+
+4. At the top of the form, select **Edit Form > Edit columns**
+
+5. In the **Edit columns** pane, check (to show) or uncheck (to hide) the box for the column or columns as needed.
+
+> Note: If you want to re-arrange the order of the columns, either drag-and-drop the column name, or select the up or down arrow next to the column name as preferred.
+
+6. When you're finished, select **Save**.
+
+## Specify conditional formula to show or hide columns
+You can show or hide columns in a list form based on another column's value by specifying a formula that are equations that perform conditional checks on values in a SharePoint list or library. 
+
+To specify a conditional formula for a column, in the **Edit columns** pane:
+
+1. Navigate to the desired column for which you want to set a conditional formula and find the more options menu (...) next to that column
+
+2. In the more options, select **Edit conditional formula**
+
+3. In the **Edit conditional formula** dialog:
+- To determine whether this column is shown or hidden, specify a conditional formula based on the value of another column.
+- You can leave it blank to clear the condition
+
+4. When you are finished, select **Save**
+
+### Get started with conditional formulas
+Formulas are equations that perform conditional expressions on column values in a list or library. A formula starts with an equal sign (=) followed by the _if_ function that returns either a _true_ or a _false_ result. 
+
+For example, the following formula checks if the column *Category*'s value is *Product Management*.
+
+```
+=if([$Category]=='Product Management','true', 'false')
+```
+
+Returning _true_ will result in hiding the column in the form while returning _false_ will not.
+
+The column is represented by specifying the **internal name** of the field surrounded by square brackets and preceeded by a dollar sign: [$InternalName]. For example, to get the value of a field with an internal name of "ProductName", use [$ProductName].
+
+#### Supported column types in conditional show or hide
+While the formula supports many of the available column types, the following column types are not yet supported:
+1. Currency columns
+2. Location columns
+3. Calculated columns
+
+#### Quick formula reference
+
+##### Choice column
+The following formula checks if the choice column [$Category] has a value *Product Management*:
+
+```
+=if([$Category]=='Product Management','true', 'false')
+```
+
+##### Number column
+The following formula checks if the choice column [$Flightscost] is less than or equal to *120*:
+
+```
+=if([$Flightscost]<=120, 'true','false')
+```
+
+You can also do arithmetic calculations such as adding the value of two columns and checking its sum, as follows in the following formula:
```

Comment:
You can also do arithmetic calculations, such as adding the value of two columns and checking its sum as follows in the following formula: